translate/Main.hs

```diff
@@ -32,6 +33,12 @@ data DynSignal = Signal Int deriving Eq
 
 instance Show DynSignal where show (Signal i) = [chr (ord 'A' + i)]
 
+instance Ord DynSignal
+    where
+        (<=) (Signal x) (Signal y)
```

Comment:
I actually just forgot to change this to compare, so I'm gonna use that. It just seems a little nicer to me for some reason